### PR TITLE
fix boolean and integer conversion

### PR DESCRIPTION
**Pull request by ipa-jba**
boolean and integer values were serialized as doubles (`True` or `1` to `1.0` and `False` or `0` to `0.0`) which breaks parsing the generated dictionaries with the c++ yaml module..

### CODE REVIEW

File: src/rospy_message_converter/message_converter.py [hunk 1]
```diff
@@ -93,6 +94,10 @@ def _convert_to_ros_type(field_type, field_value):
         field_value = _convert_to_ros_time(field_type, field_value)
     elif field_type in ros_primitive_types:
         field_value = _convert_to_ros_primitive(field_type, field_value)
+    elif field_type in ros_integer_types:
+        field_value = int(field_value)
+    elif field_type in ros_boolean_types:
+        field_type = bool(field_type)
```
Comment (mintar) [inline src/rospy_message_converter/message_converter.py]:
```suggestion
        field_value = bool(field_value)
```

File: src/rospy_message_converter/message_converter.py [hunk 2]
```diff
@@ -156,6 +161,10 @@ def _convert_from_ros_type(field_type, field_value):
         field_value = _convert_from_ros_time(field_type, field_value)
     elif field_type in ros_primitive_types:
         field_value = field_value
+    elif field_type in ros_integer_types:
+        field_value = int(field_value)
+    elif field_type in ros_boolean_types:
+        field_type = bool(field_type)
```
Comment (mintar) [inline src/rospy_message_converter/message_converter.py]:
```suggestion
        field_value = bool(field_value)
```

**Comment by mintar** [comment]
Thanks for your contribution! I've tried to reproduce the faulty behaviour, but I couldn't. Could you add some test cases to `test/*.py` that fail before your fix and succeed afterwards?

**Comment by mintar** [comment]
@ros-pull-request-builder retest this please

**Comment by ipa-jba** [comment]
actually I cannot reproduce it either and given that I will close this PR for now

**Pull request closed without merging**